package.json

```
@@ -33,6 +32,8 @@
     "@types/node": "^14.14.43",
     "browser-or-node": "^1.3.0",
     "crypto-ld": "^6.0.0",
+    "dcc-context": "digitalcredentials/dcc#rc-1.0.1",
```

```suggestion
    "@digitalcredentials/dcc-context": "^1.0.0",
```
The published package is ready now - let's use that.